functions-and-scope.js

```diff
@@ -17,6 +17,24 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 6
 
 
+
+// ------- Uitwerking 1a ---------------------------------------------------
+
+//Pseudo code:
+//* Declareer een for-loop om de array te checken
+//* Maak gebruik van de .lenght property
+//* Geef de conditie mee waarop gechecked moet worden
+//* Declareer een count variabele om de cum laude cijfers in bij te houden
+
+let output = 0;
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+    output= output + 1;
```

Comment:
Je zou dit korter kunnen schrijven:
```suggestion
    output++;
```